components/ReadMoreIn.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div class="read-more-in l-container l-container--14col">
+    <div class="read-more-in-text">
+      Read More In:
+    </div>
+    <ul class="read-more-in-links">
+      <li class="read-more-in-links-item">
+        <a href="/news">News</a>
+      </li>
+      <li class="read-more-in-links-item">
+        <a href="/arts-entertainment/">Arts & Entertainment</a>
+      </li>
+      <li class="read-more-in-links-item">
+        <a href="/food">Food</a>
+      </li>
+      <li class="read-more-in-links-item">
+        <a href="/election-2021">Election 2021</a>
+      </li>
```

Comment:
I don't really love hardcoding the list of links here, but if this component isn't used elsewhere it kind of makes sense...  What about keeping the list in the component but putting it in the component's `data`?, just thinking about updating the list or changing the component in the future.

Just a suggestion though. Your call.